cmd/tools/migration/migration/runner.go

```diff
@@ -82,8 +82,7 @@ func (r *Runner) initEtcdCli() {
 
 func (r *Runner) init() {
 	r.initEtcdCli()
-
-	r.session = sessionutil.NewSession(
+	r.session = sessionutil.NewSessionWithETCD(
```

Comment:
Use the component name consistently, do not use ETCD, change it to Etcd is better 🤔.